docs/_data/navs.yml

```diff
@@ -17,6 +17,8 @@ v1.0.x:
   - label: Pipeline hooks
   - label: The deployment context
   - label: Creating a plugin pack
+  - label: Creating an in-repo-addon plugin
```

Comment:
> Alternatively we could look at how to separate label and link

Yep, if you check out the next line, that is what I did.

I like `Creating in-repo plugins` though.